doc/man1/openssl-pkcs12.pod.in

```diff
@@ -132,6 +137,11 @@ and so the input is just verified.
 =item B<-legacy>
 
 Use legacy mode of operation and automatically load the legacy provider.
+If OpenSSL is not installed system-wide,
+it is necessary to also use, for example, B<-provider-path ./providers>
```

Comment:
Sorry for nitpicking: `C<-provider-path ./providers>`

Reply:
All right, changed.